### PR TITLE
ejabberd_oauth language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/ejabberd_oauth.erl [hunk 1]
```diff
@@ -547,7 +547,7 @@ process(_Handlers,
                     ?C(<<" is maintained by ">>),
                     ?XAC(<<"a">>,
                          [{<<"href">>, <<"https://www.process-one.net">>},
-                          {<<"title">>, <<"ProcessOne - Leader in Instant Messaging and Push Solutions">>}],
+                          {<<"title">>, <<"ProcessOne—Leader in Instant Messaging and Push Solutions">>}],
```
Comment (comradekingu) [inline src/ejabberd_oauth.erl:550]:
```suggestion
                          {<<"title">>, <<"ProcessOne—Instant Messaging and Push Solutions">>}],
```

Comment (badlop) [reply src/ejabberd_oauth.erl:550]: Your next step should be something like
```diff
- 2001 - A Space Oddyssey
+ 2001 - A Guy Takes a Peaceful Walk on Sky
```

**Comment by badlop** [comment]
Everything else looks perfect. Thanks for reporting! It will also be updated in the documentation page https://docs.ejabberd.im/developer/ejabberd-api/admin-api/

File: src/ejabberd_oauth.erl (hunk 1 above)
Comment (Neustradamus) [reply src/ejabberd_oauth.erl:550]: I have discovered the problem here.
Better to revert this part.

Comment (comradekingu) [reply src/ejabberd_oauth.erl:550]: The problem being?

Comment (Neustradamus) [reply src/ejabberd_oauth.erl:550]: Original:
- "ProcessOne - Leader in Instant Messaging and Push Solutions"

Your version:
- "ProcessOne—Leader in Instant Messaging and Push Solutions"

Comment (comradekingu) [reply src/ejabberd_oauth.erl:550]: That is an em dash. I meant to do that, if it makes a difference.

**Pull request merged**